src/ui/pages/AssetInfo/AssetTitleAndChart.tsx

```diff
@@ -1,93 +1,221 @@
+    handleRangeSelect({ endRangeIndex: null, startRangeIndex: null });
+  }, [chartPoints, handleRangeSelect]);
 
   return (
-    <VStack gap={12}>
-      <HStack gap={8} alignItems="center">
-        <div style={{ position: 'relative' }}>
-          <TokenIcon
-            src={asset.iconUrl}
-            symbol={asset.symbol}
-            size={40}
-            title={asset.name}
-          />
-          {asset.new ? (
-            <UIText
-              kind="caption/accent"
-              color="var(--white)"
-              style={{
-                position: 'absolute',
-                top: 32,
-                left: 4,
-                background: 'var(--black)',
-                borderRadius: 6,
-                padding: '2px 4px',
-              }}
-            >
-              new
-            </UIText>
-          ) : null}
-        </div>
-        <VStack gap={0}>
-          <UIText kind="caption/regular" color="var(--neutral-500)">
-            {asset.symbol}
-          </UIText>
-          <HStack gap={4} alignItems="center">
-            <UIText kind="headline/h3" style={{ display: 'flex' }}>
-              {asset.name}
+    <VStack gap={16}>
+      <VStack gap={12}>
+        {isUntrackedAsset ? (
+          <VStack gap={0}>
+            <UIText kind="headline/hero">Price Not Tracked</UIText>
+            <UIText kind="body/regular">
+              <TextAnchor
+                href={REQUEST_TOKEN_LINK}
+                rel="noopener noreferrer"
+                target="_blank"
+                style={{ color: 'var(--primary)', cursor: 'pointer' }}
+              >
+                Let us know
+              </TextAnchor>{' '}
+              if you’d like to see it on Zerion
             </UIText>
-            {asset.verified ? <VerifiedIcon /> : null}
-          </HStack>
-        </VStack>
-      </HStack>
-      {isUntrackedAsset ? (
-        <VStack gap={0}>
-          <UIText kind="headline/hero">Price Not Tracked</UIText>
-          <UIText kind="body/regular">
-            <TextAnchor
-              href={REQUEST_TOKEN_LINK}
-              rel="noopener noreferrer"
-              target="_blank"
-              style={{ color: 'var(--primary)', cursor: 'pointer' }}
-            >
-              Let us know
-            </TextAnchor>{' '}
-            if you’d like to see it on Zerion
-          </UIText>
-        </VStack>
-      ) : (
-        <HStack gap={8} alignItems="end">
-          <UIText kind="headline/hero">
-            {formatPriceValue(asset.meta.price ?? 0, 'en', currency)}
-          </UIText>
-          {asset.meta.relativeChange1d != null ? (
+          </VStack>
+        ) : (
+          <VStack gap={0}>
+            <HStack gap={8} alignItems="end">
+              <UIText
+                kind="headline/hero"
+                ref={priceElementRef}
+                color="transparent"
+              >
+                {formatPriceValue(asset.meta.price || 0, 'en', currency)}
+              </UIText>
```

Comment:
It seems that setting content of this element during react's render is unnecessary. Content is always updates anyway in `handleRangeSelect`, even for the initial render, right?